includes/site-health/audit-autoloaded-options/hooks.php

```diff
@@ -26,3 +26,86 @@ function perflab_aao_add_autoloaded_options_test( $tests ) {
 	return $tests;
 }
 add_filter( 'site_status_tests', 'perflab_aao_add_autoloaded_options_test' );
+
+/**
+ * Register admin actions for handling autoload enable/disable.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_register_admin_actions() {
+	add_action( 'admin_action_perflab_aao_update_autoload', 'perflab_aao_handle_update_autoload' );
+}
+add_action( 'admin_init', 'perflab_aao_register_admin_actions' );
+
+/**
+ * Callback for handling disable autoload action.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_handle_update_autoload() {
+	// Verify nonce.
+	$nonce = isset( $_GET['_wpnonce'] ) ? sanitize_text_field( $_GET['_wpnonce'] ) : '';
+
+	if ( ! wp_verify_nonce( $nonce, 'perflab_aao_update_autoload' ) ) {
+		wp_die( esc_html__( 'Nonce verification failed.', 'performance-lab' ) );
+	}
+
+	$option_name = isset( $_GET['option_name'] ) ? sanitize_text_field( $_GET['option_name'] ) : '';
+	$autoload    = isset( $_GET['autoload'] ) ? sanitize_text_field( $_GET['autoload'] ) : '';
+	$value       = isset( $_GET['value'] ) ? sanitize_text_field( $_GET['value'] ) : '';
+
+	if ( ! current_user_can( 'manage_options' ) ) {
+		wp_die( esc_html__( 'Permission denied.', 'performance-lab' ) );
+	}
+
+	if ( empty( $option_name ) || empty( $autoload ) ) {
+		wp_die( esc_html__( 'Invalid option name or autoload value.', 'performance-lab' ) );
+	}
+
+	// Check if the option exists.
+	if ( false === get_option( $option_name ) ) {
+		// Option doesn't exist, return an error or handle the situation accordingly.
+		wp_die( esc_html__( 'The option does not exist.', 'performance-lab' ) );
+	}
+
+	$result = wp_set_option_autoload( $option_name, $autoload );
+
+	if ( $result ) {
+		// Update modified options list.
+		$modified_options = get_option( 'perflab_aao_modified_options', array() );
+		if ( 'yes' === $autoload && isset( $modified_options[ $option_name ] ) ) {
+			unset( $modified_options[ $option_name ] );
+		} else {
+			$modified_options[ $option_name ] = $value;
+		}
+		update_option( 'perflab_aao_modified_options', $modified_options );
+
+		wp_safe_redirect( admin_url( 'site-health.php?autoload_updated=true' ) );
+		exit;
```

Comment:
To facilitate unit testing:

```suggestion
		if ( wp_safe_redirect( admin_url( 'site-health.php?autoload_updated=true' ) ) ) {
			exit;
		}
```